Prac_06/guitar.py

```diff
@@ -0,0 +1,24 @@
+
+"""
+Word Occurrences
+Estimate: 10 minutes
+Actual:   5 minutes
+"""
+class Guitar:
+
+    def __init__(self,name="",year=0,cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+
+    def get_age(self):
+        age = 2023 - self.year
+        return age
+
+    def is_vintage(self):
+        age = self.get_age()
+        if age >= 50:
```

Comment:
if the only usage for get_age is to determine if its vintage include it within the is_vintage function